Content.Server/GameTicking/GameTicker.cs

```diff
@@ -203,7 +206,7 @@ public void RestartRound()
                 if (PlayerManager.PlayerCount == 0)
                     _roundStartCountdownHasNotStartedYetDueToNoPlayers = true;
                 else
-                    _roundStartTimeUtc = DateTime.UtcNow + LobbyDuration;
+                    _roundStartTimeTicks = IoCManager.Resolve<IGameTiming>().RealTime.Ticks + LobbyDuration.Ticks;
```

Comment:
Again, you have an `IGameTiming` dependency so use that instead of resolving every time. And don't use ticks please.